template/deployments/helm/charts/templates/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: envvars-configmap
+data:
+  envVariables: {{ print "{{ .Values.envVariables }}" }}
```

Comment:
This shouldnt have the `envVariables` key, it should put the values.yaml reference directly here